Src/HostingStartup/HostingStartup.Net45/FileDiagnosticsTelemetryModule.cs

```diff
@@ -0,0 +1,223 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.HostingStartup
+{
+    using System;
+    using System.Diagnostics;
+    using System.Diagnostics.Tracing;
+    using System.Globalization;
+    using System.IO;
+    using System.Security;
+    using System.Security.Principal;
+
+    using Microsoft.ApplicationInsights.Extensibility.Implementation.Tracing;
+    
+    /// <summary>
+    /// Diagnostics telemetry module for azure web sites.
+    /// </summary>
+    public class FileDiagnosticsTelemetryModule : IDisposable, ITelemetryModule
+    {
+        private readonly WindowsIdentity currentIdentity = WindowsIdentity.GetCurrent();
```

Comment:
Also pre-extract the name of the identity as you only use `.Name` property when logging exceptions. Lazy initialization may also help performance